src/tests/server.rs

```diff
@@ -57,17 +57,25 @@ fn block_traffic_via_arbitrary_header_and_value() {
         CrateBuilder::new("dl_no_ua", user.as_model().id).expect_build(conn);
     });
 
-    let mut req = anon.request_builder(Method::Get, "/api/v1/crates/dl_no_ua/0.99.0/download");
+    let mut req = anon.request_builder(Method::GET, "/api/v1/crates/dl_no_ua/0.99.0/download");
     // A request with a header value we want to block isn't allowed
-    req.header("User-Agent", "1");
-    req.header("X-Request-Id", "abcd"); // Needed for the error message we generate
+    req.header(header::USER_AGENT, "1");
+    req.header(
+        // Needed for the error message we generate
+        // FIXME: Simplify once we hit `conduit-test 0.9.0-alpha.2`
+        header::HeaderName::from_lowercase(b"x-request-id").unwrap(),
```

Comment:
It'd be nice the http crate had that header..

Reply:
FYI: https://github.com/hyperium/http/pull/411

Reply:
Awesome!  Thanks for opening the upstream PR.